snap/local/opt/kafka/bin-wrapper.bash

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -eu
+
+${SNAP}/bin/${bin_script} "${@}"
```

Comment:
Is this path correct?

Reply:
**question** comparing to the previous version, why not impositing if not specified:

```
LOG_DIR=`$SNAP_COMMON/var/log/kafka`
```

and 

```
JAVA_HOME/PATH
```

?

These seem to be appropriate for this script as, anyway, anyone using the bash app of the Kafka snap should not be messing around with these

Reply:
Ah, for LOG_DIR, maybe one can put a formulation, that we set these variables if they have not been already, like
```
if [ ! -z $LOG_DIR ]
```
This way, if really wants, the user could override this eventually

Reply:
See other comment. Kafka will unpack into the root filesystem, and so `bin` is where the commands unpack to as it's part of the package. 
LOG_DIR and JAVA_HOME is hard-coded in the `snapcraft.yaml`, so anyone calling the snap services / apps will have these already set.